### ISSUE TITLE
"angular" in window check is wrong

### ISSUE THREAD

**Issue by kamtschatka** [JetBrains/svg-mixer#77]
See https://github.com/JetBrains/svg-mixer/blob/a4da6b33143281d3ba433d20ad81b727de35037e/packages/svg-mixer/runtime/src/browser-sprite.js#L101

The code '"angular" in window' will return true if window.angular is undefined, but it has a getter/setter defined. 

Here an example:
```
var myWindow = {};
Object.defineProperty(myWindow, "angular", {
  get : function () {}
});
console.log("angular" in myWindow);
console.log(myWindow.angular);
console.log(typeof window.angular);
```

This leads to an exception down the road, when locationChangeAngularEmitter is being checked and functions on window.angular are being invoked.

**Comment by kisenka** [comment]
@kamtschatka could you please make a PR in branch `v1`?

**Comment by kisenka** [comment]
@kamtschatka or maybe suggest a solution?

**Comment by kamtschatka** [comment]
I have opened a PR to master: https://github.com/JetBrains/svg-mixer/pull/78
I am not sure if that PR in branch v1 is also needed? And what about the v2 branch?
Let me know which PRs i should create, I will get to it, when i have some time.

**Comment by kisenka** [comment]
Yep, could you please make PR at v1 branch. Updated version of runtime will be ready soon.

**Comment by kamtschatka** [comment]
Should be done: https://github.com/JetBrains/svg-mixer/pull/83

**Comment by kisenka** [comment]
Fixed in `svg-sprite-loader@4.2.3`
